python/py-mapclassify/Portfile

```diff
@@ -24,10 +24,13 @@ checksums           rmd160  df017a4a0f924f0418f367a0410633f4e31e50fe \
                     sha256  4441798d55a051e75206bf46dccfc8a8f8323aac8596d19961d11660c98677ca \
                     size    3577980
 
-python.versions     37 38 39 310 311
+python.versions     39 310 311
```

Comment:
are there no dependents for the PY37 and PY38 subports?

Reply:
> are there no dependents for the PY37 and PY38 subports?

Ahh, there is indeed for py38: `py38-geoplot`.
@Schamschula Shall we keep the support for Python 3.8?

Reply:
Python 3.8 is still supported so the answer is very likely going to be "Yes". Please make that change by pinning thy PY38 subport to its latest supported version.

Reply:
I put back Python 3.8.